content/memory_model.tex

```diff
@@ -60,6 +60,9 @@ \subsection{Atomicity Guarantees}\label{subsec:amo_guarantees}
 guarantee that concurrent accesses by any of these routines to the same
 location and using the same datatype (specified in Tables~\ref{stdamotypes} and
 \ref{extamotypes}) will be exclusive.
+Exclusivity is also guaranteed when the target \ac{PE} performs a wait or test
+operation on the same location and with the same datatype as one or more atomic
+operations.
```

Comment:
This change might require implementing wait/test as an atomic operation.

Reply:
Given that users already use AMOs with wait/test (e.g. using an ivar as a counter), should we consider this to be a change?